src/lib/cache_dir/cache_dir.ml

```diff
@@ -1,10 +1,19 @@
 open Core
+open Async
 
 let autogen_path = Filename.temp_dir_name ^/ "coda_cache_dir"
 
 let manual_install_path = "/var/lib/coda"
 
-let brew_install_path = "/usr/local/var/coda"
+let brew_install_path =
+  match
+    Thread_safe.block_on_async_exn (fun () ->
+        Process.run ~prog:"brew" ~args:["--prefix"] () )
+  with
+  | Ok brew ->
+      brew ^ "/var"
+  | _ ->
+      "/usr/local/var"
```

Comment:
Just noting this is different than the original string

Reply:
Yeah I messed that up, thx